go/vt/vtgate/engine/fake_vcursor_test.go

```diff
@@ -41,6 +42,11 @@ func (t noopVCursor) Context() context.Context {
 	return context.Background()
 }
 
+func (t noopVCursor) SetContextTimeout(timeoutMilli int) context.CancelFunc {
+	_, cancel := context.WithTimeout(t.Context(), (time.Duration(timeoutMilli) * time.Millisecond))
```

Comment:
Not terribly important but it seems better to just make this a noop as well and just return a noop "cancel" function.